base/operators.jl

```diff
@@ -1149,40 +1149,63 @@ julia> filter(!isletter, str)
 !(f::ComposedFunction{typeof(!)}) = f.inner #allows !!f === f
 
 """
-    Fix1(f, x)
+    Fix{n}(f, x)
+    Fix{kw}(f, x)
+    Fix(f; [kw]=x)
```

Comment:
```suggestion
    Fix(f; [kw=]x)
```
`Fix(f; =x)` is not valid, `Fix(f; x)` is.

Reply:
Maybe we should write this as `Fix(f; kw=x)` instead, since it will imply that `Fix{kw}(f, x)` in the preceding line is the same as `Fix(f; kw=x)`. And also perhaps `Fix{:kw}(f, x)` over `Fix{kw}(f, x)`?

Reply:
Actually I'll just go with your suggestion. It's not clear how to indicate this subtle point in the signature alone, I think people will just read the docs.

Reply:
I also prefer `Fix(f; kw=x)`. Keyword argument names everywhere are optional, so this is not a property of Fix specifically and should not figure into the Fix docstring.

Reply:
Done :+1: